lib/benchee/formatters/console.ex

```diff
@@ -79,23 +90,35 @@ defmodule Benchee.Formatters.Console do
   ## Examples
 
   ```
-  iex> jobs = %{ "My Job" =>%Benchee.Statistics{average: 200.0, ips: 5000.0,std_dev_ratio: 0.1, median: 190.0}, "Job 2" => %Benchee.Statistics{average: 400.0, ips: 2500.0, std_dev_ratio: 0.2, median: 390.0}}
-  iex> Benchee.Formatters.Console.format_jobs(jobs, %{comparison: false, unit_scaling: :best})
+  iex> scenarios = [
+  ...>   %Benchee.Benchmark.Scenario{
+  ...>     job_name: "My Job", run_time_statistics: %Benchee.Statistics{
+  ...>       average: 200.0, ips: 5000.0,std_dev_ratio: 0.1, median: 190.0
+  ...>     }
+  ...>   },
+  ...>   %Benchee.Benchmark.Scenario{
+  ...>     job_name: "Job 2", run_time_statistics: %Benchee.Statistics{
+  ...>       average: 400.0, ips: 2500.0, std_dev_ratio: 0.2, median: 390.0
+  ...>     }
+  ...>   }
+  ...> ]
+  iex> Benchee.Formatters.Console.format_scenarios(scenarios, %{comparison: false, unit_scaling: :best})
   ["\nName             ips        average  deviation         median\n",
   "My Job        5.00 K      200.00 μs    ±10.00%      190.00 μs\n",
   "Job 2         2.50 K      400.00 μs    ±20.00%      390.00 μs\n"]
```

Comment:
Just love how these tests show that nothing has changed for the output :heart_eyes: